pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "azimuth"
-version = "2.6.1"
+version = "2.6.2"
```

```suggestion
version = "2.7.0"
```
As discussed offline!

---

Yup! Sorry. I bumped the minor version directly on `main`: e888bbf576d73902f16adcd91bd2176d719ee9b2.